pkg/server/controllers.go

```diff
@@ -615,12 +616,22 @@ func (s *Server) installAPIBindingController(ctx context.Context, config *rest.C
 		return err
 	}
 
+	var rootApiExportInformer apisinformers.APIExportInformer
+	var rootApiResourceSchemaInformer apisinformers.APIResourceSchemaInformer
+
+	if s.rootShardKcpSharedInformerFactory != nil {
+		rootApiExportInformer = s.rootShardKcpSharedInformerFactory.Apis().V1alpha1().APIExports()
+		rootApiResourceSchemaInformer = s.rootShardKcpSharedInformerFactory.Apis().V1alpha1().APIResourceSchemas()
+	}
+
 	c, err := apibinding.NewController(
 		crdClusterClient,
 		kcpClusterClient,
 		s.kcpSharedInformerFactory.Apis().V1alpha1().APIBindings(),
 		s.kcpSharedInformerFactory.Apis().V1alpha1().APIExports(),
+		rootApiExportInformer,
```

Comment:
would call all of these "remove....Informer" because the root shard will also need it, just inversed.

In that context, the bootstrapping has to be in a way that the root shard can start up without the remote informers being synced.